pkg/cloudprovider/providers/azure/azure_managedDiskController.go

```diff
@@ -35,56 +41,108 @@ type ManagedDiskController struct {
 	common *controllerCommon
 }
 
+// ManagedDiskOptions specifies the options of managed disks.
+type ManagedDiskOptions struct {
```

Comment:
Please add documentation for this `struct`. Specifically the difference between `zonesPresent` and `ZonesPresent`

Reply:
ACK, will do